pandas/_libs/index.pyi

```diff
@@ -50,6 +50,7 @@ class UInt32Engine(IndexEngine): ...
 class UInt16Engine(IndexEngine): ...
 class UInt8Engine(IndexEngine): ...
 class ObjectEngine(IndexEngine): ...
+class StringEngine(ObjectEngine): ...
```

Comment:
Think this needs to be updated to match the class implementation

Reply:
Updated, thanks